tests/lib/docker.py

```diff
@@ -23,8 +23,8 @@ def get_mapped_port_for_host(self, name, container_port):
         Given a port on a container return the port on the host to which it is
         mapped
         """
-        if isinstance(container_port, int):
-            container_port = f"{container_port}/tcp"
+        assert isinstance(container_port, int)
```

Comment:
We can just skip this assertion, can't we? We'll soon discover if it's being passed nonsense.

Reply:
I guess so